ui/apps/platform/src/routePaths.ts

```diff
@@ -49,7 +49,7 @@ export const integrationEditPath = `${integrationsPath}/:source/:type/edit/:id`;
 export const integrationsListPath = `${integrationsPath}/:source/:type`;
 export const listeningEndpointsBasePath = `${mainPath}/listening-endpoints`;
 export const networkBasePath = `${mainPath}/network-graph`;
-export const networkPath = `${networkBasePath}/:detailType?/:detailId?`;
+export const networkPath = `${networkBasePath}/:nodeType?/:nodeId?/:detailType?/:detailID?`;
```

Comment:
`detail_` didn't make much sense for keeping the state of the selected node. We will use `node_` instead. We can use `detail_` for the nested view.